modules/doc/content/getting_started/installation/install_mambaforge.md

```diff
@@ -0,0 +1,41 @@
+## Install Mambaforge3 id=installconda
+
+Installing Mambaforge3 is straight forward. Download, install, and configure. If you run into issues during these steps, please visit our [troubleshooting guide for Conda](troubleshooting.md#condaissues optional=True). This installation
+guide relies on the utilization of `mamba`, a C-based package manager for Conda that takes advantage of parallelization
+and is, therefore, considerable faster.   
+
+- +Linux Users:+
+
+  ```bash
+  curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+  bash Mambaforge-Linux-x86_64.sh -b -p ~/mambaforge3
+  ```
+
+- +Macintosh Users with Intel processors:+
+
+  ```bash
+  curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-x86_64.sh
+  bash Mambaforge-MacOSX-x86_64.sh -b -p ~/mambaforge3
+  ```
+
+- +Macintosh Users with ARM processors:+
+
+  ```bash
+  curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-arm64.sh
+  bash Mambaforge-MacOSX-arm64.sh -b -p ~/mambaforge3
+  ```
```

Comment:
Since we don't officially support ARM platforms on Mac (yet), I think this should be removed until we do.

Reply:
darn... yeah.. . sad panda